qsr_lib/src/qsrlib_qsrs/qsr_tpcc.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function, division
+from qsrlib_qsrs.qsr_triadic_abstractclass import QSR_Triadic_1t_Abstractclass
+import math
+
+
+import sys
+import math
+
+NUMBER_OF_PARTITIONS = 8
+PARTITION_SIZE = (2.0 * math.pi) / NUMBER_OF_PARTITIONS  
```

Comment:
What are lines 10 and 11? I mean I know that they are module constants, but should they be defined as constants instead of class member for first and a set property for second? Anyway, your call, not too fuzzy.